.github/workflows/python-package.yml

```
@@ -8,7 +8,7 @@ on:
     branches: [ develop, pre-release, master, main ]
     tags: v*
   pull_request:
-    branches: [ develop, pre-release, master, main ]
+    branches: [ develop, pre-release, master, main, enhance-torch-compile ]
```

```suggestion
    branches: [ develop, pre-release, master, main]
```
Please delete this now as this PR is ready.